Frameworks/CoreGraphics/CGBitmapContext.mm

```diff
@@ -55,6 +55,7 @@ CGImageRef CGBitmapContextCreateImage(CGContextRef context) {
 @Notes
 */
 CGBitmapInfo CGBitmapContextGetBitmapInfo(CGContextRef context) {
+#if 0
```

Comment:
Why the #if 0s? If this is because of he refactor @msft-Jeyaram is doing, we might as well remove this code and add it back as needed.  Presumably, we will still not do the context->Backing()->DestImage()->Backing()->Dosomething() anymore.